src/containers/ProfilePage/index.jsx

```diff
@@ -109,7 +111,7 @@ export default function ProfilePage() {
               <ProfileSection
                 key={section.label}
                 eventKey={index.toString()}
-                label={section.label}
+                label={t(`${section.label}`)}
```

Comment:
fix

Reply:
never mind, i see, you're substituting the translation key here i guess

Reply:
yeah, that's right